src/ucp/core/ucp_mm.h

```diff
@@ -27,6 +27,8 @@
    existing region */
 #define UCP_MM_UCT_ACCESS_MASK UCT_MD_MEM_ACCESS_ALL
 
+#define UCP_MM_UCT_NON_CACHED UCS_BIT(31)
```

Comment:
imo, such flag needs to be defined in UCT instead

Reply:
can we avoid this flag, since we already have UCP_MEMH_FLAG_NON_CACHED?